### DISCUSSION TITLE
Changing colors isn't making any changes in the app.

### DISCUSSION THREAD

**Discussion by premdasvm** [obytes/react-native-template-obytes#216] [Q&A] [answered]
Hey,

I'm trying to update the color palette for my app, and I've made changes in the colors.js file like this:

```js
primary: {
  50: "#E6F7FF",
  100: "#C2E0FF",
  200: "#9ECAFF",
  300: "#7AB4FF",
  400: "#569EFF",
  500: "#3289FF",
  600: "#0E73FF",
  700: "#0068E6",
  800: "#005EBF",
  900: "#005299",
}
```

```ts
export const buttonVariants: BVariant = {
	defaults: {
		container: "flex-row items-center justify-center rounded-2xl px-12 py-3 my-2",
		label: "text-[16px] font-medium text-white",
		indicator: "text-white h-[30px]",
	},
	primary: {
		container: "bg-primary-600",
		label: "",
		indicator: "text-white",
	},
	secondary: {
		container: "bg-primary-600",
		label: "text-secondary-600",
		indicator: "text-white",
	},
	outline: {
		container: "border border-neutral-400",
		label: "text-black dark:text-charcoal-100",
		indicator: "text-black",
	},
};
```

However, it seems like my buttons are still displaying an orange color from the default template. What am I missing here?

![Screenshot 2023-09-27 at 10 36 27 PM](https://github.com/obytes/react-native-template-obytes/assets/14838550/5940128a-7cd6-42ec-b901-1691e40b6516)

**Comment by yjose** [comment, marked as answer]
As you are uodating tailwindcss config, you need to rerun you server without  cache using 'pnpm start --c'

**Reply by premdasvm** [reply]
hi, Thanks for the quick replay. It's working now. cache issue i think.

**Comment by premdasvm** [comment]
@yjose Can you show me an example how you are styling the `FlashList` or `FlatList` using tailwind? in the template you are just returning it as normal component, cant `FlashList` be styled using `nativeWind` ?

**Reply by yjose** [reply]
You can warp your list with styled following [this format](https://www.nativewind.dev/api/styled#styling-multiple-properties),We faced some issues back then when we tried it. I am waiting for the next [NativeWind update](https://www.nativewind.dev/v4/guides/custom-components#handling-components-with-style-attribute-props), which introduces an easy way to make it work without issues. However, you can still try the old approach if you want.
